leakcanary-analyzer/src/main/java/leakcanary/LeakTrace.kt

```diff
@@ -21,17 +21,17 @@ data class LeakTrace(
     elements.dropLast(1)
         .forEachIndexed { index, leakTraceElement ->
           val currentReachability = expectedReachability[index]
-
+          val numOfSpaces = leakTraceElement.className.lastIndexOf('.') + 3
           leakInfo += """
-        |├─ ${leakTraceElement.className}
-        |│${getReachabilityString(currentReachability)}
-        |│${getPossibleLeakString(currentReachability, leakTraceElement, index)}
-        |
-        """.trimMargin()
+        #├─ ${leakTraceElement.className}
+        #│${getReachabilityString(currentReachability, numOfSpaces)}
+        #│${getPossibleLeakString(currentReachability, leakTraceElement, index, numOfSpaces)}
+        #
+        """.trimMargin("#")
```

Comment:
Why change to `#` ?

Reply:
Since `LeakTaceElement` uses the same character as the trimMargin default separator, then the pipe gets cut off.